### DISCUSSION TITLE
Issues getting started with React95

### DISCUSSION THREAD

**Discussion by OrdinaireX** [React95/React95#348] [Q&A] [answered]
Hey guys!

Just wondering if we could get a more comprehensive list of sites made with React95. It would really help me think of different ways I could use it for my portfolio.

Thanks!

**Comment by ggdaltoso** [comment, marked as answer]
We currently have a section on our README with some example. You can check it out [here](https://github.com/React95/React95#awesome-projects)

You can also check the projects relying on [React95 as a dependency](https://github.com/React95/React95/network/dependents?package_id=UGFja2FnZS00Njc3OTA4ODM%3D) as well.
